### ISSUE TITLE
Can I get some sample code as to how to generate shapes please?

### ISSUE THREAD

**Issue by Joe23232** [rennis250/processing-rs#6]
I want to use this crate but I was looking for an example of how to generate ellipse, lines etc. Can I just get some sample code so I know how to import the library and get started please?

**Comment by zyansheep** [comment]
There is an example in the examples folder.

**Comment by Joe23232** [comment]
Is this project dead by any chance?

**Comment by rennis250** [comment]
No. I just have been rather busy with work for a long time. I plan to get back to this in the coming winter months and do have some updates that I need to push.

- Rob

**Comment by Joe23232** [comment]
AH I see, thanks mate
